apps/member/src/components/common/File/File.tsx

```diff
@@ -1,24 +1,47 @@
 import { createURL } from '@clab-platforms/utils';
 
 import { SERVER_BASE_URL } from '@constants/api';
+import useToast from '@hooks/common/useToast';
 
 interface FileProps extends React.PropsWithChildren {
   href: string;
+  name: string;
 }
 
-const File = ({ children, href }: FileProps) => {
+const File = ({ href, name }: FileProps) => {
+  const toast = useToast();
+
   if (!href.startsWith(SERVER_BASE_URL)) {
     href = createURL(SERVER_BASE_URL, href);
   }
 
+  const handleClickImgLink = () => {
+    fetch(href)
+      .then((res) => res.blob())
+      .then((blob) => {
+        const href = window.URL.createObjectURL(new Blob([blob]));
+        const a = document.createElement('a');
+        a.href = href;
+        a.download = name;
+        document.body.appendChild(a);
+        a.click();
+        window.URL.revokeObjectURL(href);
+        a.remove();
```

Comment:
긴 라인은 문맥을 한번 구분해주는게 가독성에 좋아요

 ```suggestion
        const href = window.URL.createObjectURL(new Blob([blob]));
        const a = document.createElement('a');
        
        a.href = href;
        a.download = name;
        document.body.appendChild(a);
        a.click();
        window.URL.revokeObjectURL(href);
        a.remove();
```